Reworked the build process (#60)
DNNCommunity/DNN.Faq: .github/CONTRIBUTING.md

* Make sure your pull request description tags the GitHub issue ID, so it is clear what issue you have fixed
* Make sure your commit messages are in the proper format

## Building the module

> It has never been that easy to build and package a module. In Visual Studio, you will see 2 project, make sure the _build project is set as the startup project (should already be), then in the debug dropwdown you will see:

Reviewer: ```suggestion
It has never been that easy to build and package a module. In Visual Studio, you will see 2 projects. Make sure the _build project is set as the startup project (should already be). Then in the debug dropdown you will see:
```

> * Package: this packages the whole module zip ready to be installed, you will find the zip file in the `Artificats` directory.

Reviewer: ```suggestion
* Package: this packages the module into an install `zip` package, ready to be installed. You will find the `zip` file in the `Artifacts` directory.
```

> * Deploy: For historical reasons, this module source code lives in the `DesktopModules\DNN.FAQ` folder but the running copy of the test site runs off the `DesktopModules/FAQ` folder. This mode will build the source code and copy the appropriate files to their correct location to test right away on your development site.

Reviewer: ```suggestion
* Deploy: For historical reasons, the module source code lives in the `DesktopModules\DNN.FAQ` folder but the running copy of the test site runs off the `DesktopModules/FAQ` folder. This mode will build the source code and copy the appropriate files to their correct location to test right away on your development site.
```
